lib/consolidate.js

```diff
@@ -171,14 +171,17 @@ function fromStringRenderer(name) {
     options.filename = path;
 
     return promisify(cb, function(cb) {
-      readPartials(path, options, function(err) {
+      readPartials(path, options, function(err, partials) {
+        var extend = (requires.extend || (requires.extend = require('util')._extend));
```

Comment:
I don't think this is necessary, you can just use `Object.assign`.

Reply:
you sure? I wasn't sure which version needs to be supported since only extend is used in this lib

Reply:
thanks for pointing that out... I forgot that it was already used in other places.